_data/projects/ElectronClippy.yml

```diff
@@ -0,0 +1,10 @@
+name: Clippy
+desc: Electron App that records your Clipboard history and Image history
+site: https://github.com/Jarmahent/Clippy
+tags:
+- electron
+- javascript
+- react
+upforgrabs:
+  name: Help Needed
```

Comment:
@Jarmahent There is no label in your repo for this name. Please either create a new label with this name or use any existing label name